apps/ledger-live-mobile/src/newArch/features/Accounts/components/AddAccountButton/index.tsx

```diff
@@ -20,12 +20,16 @@ const StyledPressable = styled(Pressable)`
   column-gap: 12px;
 `;
 
-const AddAccountButton: React.FC = () => {
+type Props = {
+  sourceScreenName: string;
+};
+
+const AddAccountButton: React.FC<Props> = ({ sourceScreenName }) => {
   const { t } = useTranslation();
   const [isAddModalOpened, setAddModalOpened] = useState<boolean>(false);
 
   const openAddModal = () => {
-    track("button_clicked", { button: "add a new account", page: "Accounts" });
+    track("button_clicked", { button: "Add a new account", page: sourceScreenName });
```

Comment:
nice catch, should it be under the FF ?

Reply:
it's already the case it's only used inside the new AccountsList